index.md

```diff
@@ -0,0 +1,28 @@
+# The Magic Wormhole protocols
+
+This site contains all the documentation and specification of protocols related
+to Magic Wormhole, which are not specific to a single client or implementation.
+It assumes the reader is already familiar with the general Magic Wormhole concept.
+
+The most important component is the **rendezvous server**. There are two aspects
```

Comment:
maybe add "aka mailbox server" to avoid confusion (while we're still .. using both?)